Translation/en/Text/Main/CircledNumbers.txt

```diff
@@ -0,0 +1,22 @@
+//Circled numbers, because they are hard to read
```

Comment:
Should be resolution scoped `width < 1920` to be consistent

Reply:
I'll also make a proper header for this one